manage_breast_screening/mammograms/views/appointment_views.py

```diff
@@ -155,6 +157,11 @@ def form_valid(self, form):
             auditor.audit_create(note)
         else:
             auditor.audit_update(note)
+        messages.add_message(
```

Comment:
Design question, but I'm wondering if this success message is unnecessary here.

Instead of redirecting back to the top of the notes tab, we could potentially redirect the user back to the appointment details tab, where the appointment note is visible as a banner.

When I tested the current version, it wasn't obvious to me that the banner was added to the appointment details tab after submitting the form on the notes tab.

Reply:
@edwardhorsford @rivalee WDYT?